src/site/downloads/mac.markdown

```diff
@@ -8,50 +8,43 @@ description: "Installing and updating Dart on your Mac."
 
 # {{ page.title }}
 
-We recommend using Homebrew
+The Dart SDK contains the VM, analyzer, formatter, dart2js compiler,
+documentation generator, package manager, and the core libraries.
+
+We recommend using [Homebrew](#homebrew)
 to manage your Dart installation.
 Another option is [downloading Dart manually](#zip).
 
 
-## Using Homebrew {#homebrew}
+## Installing and upgrading with Homebrew {#homebrew}
 
+Homebrew is a package manager for Mac OS.
 With [Homebrew](http://brew.sh/),
 installing and updating Dart is easy.
-You can use the default location or
-[specify a custom location for Dart](#homebrew-custom-location).
-
 
 ### Installing Dart {#homebrew-install-dart}
 
-The **dart** package contains the [Dart SDK](/tools/sdk/),
-which includes the Dart VM, libraries, and command-line Dart tools such as
-[dart](/tools/dart-vm/), [dart2js](/tools/dart2js/),
-[dartanalyzer](/docs/dart-up-and-running/contents/ch04-tools-dart_analyzer.html),
-[pub](/tools/pub/), and [dartdocgen](/tools/dartdocgen/).
-Optionally, you can add two more tools:
-
-* [Dartium](/tools/dartium/):
-  A special build of Chromium that includes a Dart VM.
-  Use it to interactively test and debug Dart web apps
-  without first compiling them to JavaScript.
-* The content shell:
-  Headless Dartium for automated testing.
-
-For client-side Dart work, you might want to download all the tools:
+[Install homebrew](http://brew.sh/), and then run:
 
 {% prettify huge %}
 $ brew tap dart-lang/dart
-$ brew install dart --with-content-shell --with-dartium
+$ brew install dart
 {% endprettify %}
 
-If you're working on server-side Dart,
-all you need is `dart`:
+If you use Dart for web development work, we recommend that you also
+install Dartium and Content Shell:
 
 {% prettify huge %}
 $ brew tap dart-lang/dart
-$ brew install dart
+$ brew install dart --with-content-shell --with-dartium
 {% endprettify %}
 
+### Installing dev-channel releases
+
+To get early access to new features and fixes,
+you can use the dev-channel. Head's up: dev-channel
```

Comment:
dev-channel is OK for an adjective but not for a noun. The noun (according to https://www.dartlang.org/downloads/archive/) is "dev channel" (no hyphen).